.gitignore

```diff
@@ -4,6 +4,10 @@
 # OSX trash
 .DS_Store
 
+# Intellij files
```

Comment:
Why do you have these files?

Reply:
These are just files that Intellij Goland IDE generates. (like the Vscode files)